index.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+  
+    <script  src="js/index.js"></script>
+    Hello World, we are software developers!
+    <div id="main">
+      <h1 id="title">Dr. Norman Borlaug</h1>
+      <p>The man who saved a billion lives</p>
+      <div id="img-div">
+        <img id="image" src="./images/photo.jpg" alt="photo">
+        <figcaption id="img-caption"> Dr. Norman Borlaug posing for a photo at his office</p>
+        </figcaption>
+        </div>
+
+        <div id="tribute-info">
```

Comment:
Please use  a section tag since it is a semantic tag and gives more meaning to the webpage that the div tag